mpt_trie/src/nibbles.rs

```diff
@@ -355,6 +363,17 @@ impl UpperHex for Nibbles {
 }
 
 impl Nibbles {
+    /// Create `Nibbles` that is empty.
+    ///
+    /// Note that mean that the key size is `0` and does not mean that the key
+    /// contains the `0` [`Nibble`].
+    pub fn empty() -> Self {
```

Comment:
nit: I like `Nibbles::empty` but prefer `Nibbles::new` - it's what all other collections do, e.g `Vec::new`

Reply:
I'd agree, have a slight preference for `new()`

Reply:
Alright, I'll switch this over.